lib/Stats.js

```diff
@@ -436,9 +437,19 @@ class Stats {
 					}, Object.create(null))
 				};
 				if (showPerformance) {
-					obj.entrypoints[name].isOverSizeLimit = ep.isOverSizeLimit;
+					obj[name].isOverSizeLimit = cg.isOverSizeLimit;
 				}
 			}
+
+			return obj;
+		};
+
+		if (showEntrypoints) {
+			obj.entrypoints = fnChunkGroup(compilation.entrypoints);
+		}
+
+		if (showChunkGroups) {
+			obj.namedChunkGroups = fnChunkGroup(compilation.namedChunkGroups);
```

Comment:
Add output in the `toString` part of the stats for `namedChunkGroups`.

Reply:
Yes it makes sense to not show them multiple times in the stats toString output. You can dedupe them using a `Set`